.travis.yml

```diff
@@ -69,14 +71,21 @@ script:
   - make check
   - make deb
 
+after_success:
+  - echo "TRAVIS_TAG='$TRAVIS_TAG'"
+
+before_deploy:
+  - echo "TRAVIS_TAG='$TRAVIS_TAG'"
+
+
 # Deploy debian packages to bintray
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: echo "TRAVIS_TAG='$TRAVIS_TAG'"; beaver bintray upload -N pkg/deb/*.deb
     skip_cleanup: true
     on:
         tags: true
-        condition: >
-            "$TRAVIS_OS_NAME" = "linux" -a
-            "$CC" = "gcc"
+        condition: '"$TRAVIS_OS_NAME" = "linux" && "$CC" = "gcc"'
```

Comment:
AFAICS because travis wraps the condition in within double brackets and thus expects `&&` as comparison operator.